### ISSUE TITLE
Can't open the schematic

### ISSUE THREAD

**Issue by xfce** [fishpepper/uSKY#1]
@fishpepper ,
hi, I download the doc, but can't get the detailed schematic use the latest Kicad, some IC symbols missed 
it tells not found libraries.
would you please give a pdf version schematic, thanks

**Comment by esden** [comment]
I also wanted to take a look at the files. The uSKY-cache.lib file is missing. It is necessary to commit that file with every KiCAD project unfortunately. :)

**Comment by fishpepper** [comment]
You do not need this file, just add the kicad_misc folder to the libs manually.

**Comment by esden** [comment]
Yes I saw that you have it in there as a submodule. `git submodule init` and `git submodule update` makes the schematic open fine. In general though it is KiCAD convention to include the cache file to allow for schematic symbol recovery. Especially if you ever change a symbol in the library for a different project it is important to be able to recover the original symbol. Unfortunately schematics do not keep the symbols inside the schematic file, the way it is done for footprints in the kicad_pcb files. But my problem is solved. Thanks... :D

**Comment by esden** [comment]
Here is the reason why you want to include the cache file. The KiCad libraries changed the capacitor and resistor symbols. Because the cache file is not in place it takes automatically the symbol from the library instead from the cache. The problem will be solved with KiCad 6.0 and the new file format but for now it is unfortunately necessary to include the -cache.lib files.

![screenshot from 2018-01-03 18-09-04](https://user-images.githubusercontent.com/17334/34531077-666c3166-f0b1-11e7-8391-95853363bf48.png)

**Comment by fishpepper** [comment]
I do not like to include cache files in the github repo ;) For my later designs I moved the cap and resistor to the kicad_misc submodule because of this. Seems like I did not update the usky project to use those.
